test/e2e/nodefeature/nodefeature.go

```diff
@@ -104,6 +104,9 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	SidecarContainers = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("SidecarContainers"))
 
+	// Sig-node: add e2e tests for KEP-4191
```

Comment:
So this feature can only be tested on nodes with "separate discs"?

If that's true, then it would be a better description of the this feature tag than just the reference to the KEP.

Reply:
Sure. We can follow-up with a change to improve the description. No worries.

Reply:
You've got a point. I'll make a follow-up PR later.

Reply:
@bitoku, thank you!

We need two commits:

1. Update the comment with test requirements, etc.
2. Sort the variables in an alphabetical order (per the comment in the file; something we didn't do)

Reply:
I made a PR.
https://github.com/kubernetes/kubernetes/pull/126181